workshop/Lab1/README.md

```diff
@@ -10,13 +10,17 @@ If you haven't already:
 2. Provision a cluster:
 
    ```$ ibmcloud cs cluster-create --name <name-of-cluster>```
+   
+   (If the above command doesn't work, please try:`$ ibmcloud cs cluster create classic --name <name-of-cluster>`)
 
 Once the cluster is provisioned, the kubernetes client CLI `kubectl` needs to be
 configured to talk to the provisioned cluster.
 
 1. Run `$ ibmcloud cs cluster-config <name-of-cluster>`, and set the `KUBECONFIG`
    environment variable based on the output of the command. This will
    make your `kubectl` client point to your new Kubernetes cluster.
+   
+   (If the above command doesn't work, pls try `$ ibmcloud cs cluster config --cluster <name-of-cluster>`)
```

Comment:
I need to follow up with the plugin writer and figure out what they were thinking.

Reply:
user studies apparently, https://uxdesign.cc/user-experience-clis-and-breaking-the-world-baed8709244f